docs/config-schema.md

```diff
@@ -380,6 +381,7 @@ where applicable:
   intervalSeconds: 10
   globalDimensions: 
   sendMachineID: false
+  cluster: 
```

Comment:
The indent on some of the lines of this example seem off?

Reply:
It seems ok looking at it here in Github (and in my text editor).  This is basically a dump of the "default" config.